x-pack/test/functional/es_archives/infra/alerts_test_data/mappings.json

```diff
@@ -13,6 +13,10 @@
           "ignore_above": 256,
           "type": "keyword"
         },
+        "host.name": {
```

Comment:
Would it make sense to also test a field that is excluded by the `exclude_list`?

Reply:
Makes sense. Adjusted the test data and tests.